.github/workflows/test.yaml

```diff
@@ -4,7 +4,9 @@ on:
   pull_request:
     branches:
       - main
-
+  push:
+    branches:
+      - main
```

Comment:
sneaky change, I think it should be in separate PR

Reply:
Removed